ui/app/models/mfa-login-enforcement.js

```diff
@@ -109,7 +109,7 @@ export default class MfaLoginEnforcementModel extends Model {
   }
 
   iconForMount(type) {
-    const mountableMethods = methods();
+    const mountableMethods = ALL_ENGINES.filter((engine) => !engine.requiresEnterprise);
```

Comment:
same issue here as above, make sure we keep behavior consistent and iterate over auth methods, not auth and secret.